sass/layout/_sidebar-left.scss

```diff
@@ -29,7 +29,7 @@
     position: fixed;
     left: 0;
     height: 100%;
-    z-index: 12;
+    z-index: 16;
```

Comment:
We need to include the legacy sidebar as well https://github.com/mattermost/mattermost-webapp/pull/5622/files#diff-d921a34eb9bcfe6512043c12cc85fa21L11

Reply:
I can use the same `z-index` as it displayed alternately.

Reply:
Done!